api/db-cdr/generate-cdr/make-bq-cb-menu.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# This generates the big query de-normalized tables for dataset builder.
+
+set -ex
+
+export BQ_PROJECT=$1  # project
+export BQ_DATASET=$2  # dataset
+
+# Create bq tables we have json schema for
+schema_path=generate-cdr/bq-schemas
+create_tables=(cb_criteria_menu cb_menu)
```

Comment:
So, `cb_criteria_menu` is for the current implementation and will eventually go away, and `cb_menu` is for when we separate source and standard, correct?

Reply:
correct.